servers/publikator/__tests__/document.subscriptions.int.jest.js

```diff
@@ -0,0 +1,281 @@
+const { Instance } = require('@orbiting/backend-modules-test')
+
+const {
+  Users
+} = require('@orbiting/backend-modules-auth/__tests__/auth')
+const {
+  deleteRepos
+} = require('./helpers')
+const {
+  commit,
+  PUBLISH_MUTAION
+} = require('./graphqlQueries.js')
+const docs = require('./documents')
+const {
+  createUsers
+} = require('@orbiting/backend-modules-test')
+
+beforeAll(async () => {
+  await Instance.init({
+    serverName: 'publikator'
+  })
+  await deleteRepos()
+  const { pgdb } = global.instance.context
+  await pgdb.public.users.insert(Users.Editor)
+  global.testUser = Users.Editor
+}, 1000 * 60 * 2)
+
+afterAll(async () => {
+  await global.instance.closeAndCleanup()
+  await deleteRepos()
+}, 60000)
```

Comment:
```suggestion
}, 1000 * 60)
```

(In lieu of `beforAll` timeout)